src/core/css/SearchResult.scss

```diff
@@ -23,7 +23,15 @@
 }
 
 .SearchResult-main {
+  display: flex;
+  flex-direction: row;
   flex-grow: 1;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.SearchResult-text {
+  width: 90%;
```

Comment:
I tried using pixels but it fails below a certain browser width. Should I use a media query for that?